icons/canonical.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Canonical</title><path d="M24 12c0 6.627-5.373 12-12 12-6.628 0-12-5.373-12-12C0 5.372 5.372 0 12 0c6.627 0 12 5.372 12 12zM12 2.92A9.08 9.08 0 002.92 12 9.08 9.08 0 0012 21.08 9.08 9.08 0 0021.081 12 9.08 9.08 0 0012 2.92zm0 16.722A7.64 7.64 0 014.36 12 7.64 7.64 0 0112 4.36 7.64 7.64 0 0119.641 12a7.64 7.64 0 01-7.64 7.641z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Canonical</title><path d="M19.573 0H4.427v24h15.146V0Zm-5.506 17.733a1.569 1.569 0 1 1-.001 3.137 1.569 1.569 0 0 1 .001-3.137Zm-2.64 1.559a3.82 3.82 0 0 1-2.604-1.986c-.419.19-.888.248-1.341.165a5.103 5.103 0 0 0 3.675 3.08c.367.079.741.117 1.114.115a2.246 2.246 0 0 1-.459-1.311 4.172 4.172 0 0 1-.385-.063Zm4.822-.568v.001-.001c.488-.614.831-1.34.997-2.113a5.14 5.14 0 0 0-.74-3.905c-.18.425-.487.785-.882 1.032a3.843 3.843 0 0 1-.125 3.819c.369.302.629.709.75 1.167ZM7.89 13.688a1.569 1.569 0 1 1-.001 3.137 1.569 1.569 0 0 1 .001-3.137Zm6.531-3.438a1.57 1.57 0 0 1 1.568 1.568 1.569 1.569 0 1 1-1.568-1.568Zm-6.652 2.758.12-.003a2.258 2.258 0 0 1 1.189.338 3.827 3.827 0 0 1 3.079-1.647 2.264 2.264 0 0 1 .476-1.271 5.156 5.156 0 0 0-4.864 2.583Z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Canonical</title><path d="M4.427 0v24h15.146V0H4.427zm9.994 10.25a1.568 1.568 0 1 1 0 3.136 1.568 1.568 0 0 1 0-3.136zm-1.788.175a2.268 2.268 0 0 0-.476 1.271 3.825 3.825 0 0 0-3.08 1.647 2.243 2.243 0 0 0-1.308-.335 5.155 5.155 0 0 1 4.864-2.583zm-6.31 4.831a1.568 1.568 0 1 1 3.135 0 1.568 1.568 0 0 1-3.135 0zm4.834 5.295a5.105 5.105 0 0 1-3.675-3.08c.453.083.922.025 1.34-.165a3.818 3.818 0 0 0 2.605 1.986c.127.027.256.048.385.063.01.476.17.932.459 1.31a5.157 5.157 0 0 1-1.114-.114zm2.91.318a1.568 1.568 0 1 1 0-3.136 1.568 1.568 0 0 1 0 3.136zm3.178-4.258a5.137 5.137 0 0 1-.996 2.113 2.25 2.25 0 0 0-.75-1.167 3.837 3.837 0 0 0 .125-3.818c.395-.248.702-.608.882-1.033a5.14 5.14 0 0 1 .74 3.905z"/></svg>
```
Perhaps this might fix it.